content/storage/bind-mounts.md

```diff
@@ -266,6 +266,39 @@ $ docker container stop devtest
 $ docker container rm devtest
 ```
 
+## Recursive mounts
```

Comment:
This option is only supported in the `--mount` variant, not in the shorthand `-v` / `--volume` syntax; do we need to put it inside a "tab" ?